### PR TITLE
chip, chip with cross and more

### PR DESCRIPTION
**Pull request by murliDN**

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -29,18 +29,31 @@ export default function App() {
...
-          onPress={() => Alert.alert('Primary Button pressed')}
+          onPress={() => Alert.alert("Secondary Button pressed")}
         />
         <Text>{"\n"}</Text>
         <TextButton
           title="Text Button"
-          onPress={() => Alert.alert('text Button pressed')}
+          onPress={() => Alert.alert("text Button pressed")}
+        />
+        <Chips label="Chip" />
```
Comment (jtibrewal09) [inline App.js:45]: rename chips to chip

File: src/components/chip.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//chip with image
+
+const Chips = (props) => {
+  const { label, onPress, style, showCross, showMore } = props;
+
+  return (
+    <View style={[styles.chip, style.chip]}>
+      <Text style={[styles.text, style.text]}>{label}</Text>
+      {(showCross || showMore) && (
+        <ImageButton style={{ marginLeft: 8 }} onPress={onPress}>
```
Comment (jtibrewal09) [inline src/components/chip.js:18]: Try avoiding inline styling.

Comment (jtibrewal09) [inline src/components/chip.js:18]: Use flex box CSS property for responsiveness.

Comment (murliDN) [reply src/components/chip.js:18]: done

**Pull request merged**